README.md

```diff
@@ -1,22 +1,22 @@
 # Amundsen Databuilder
 
 [![PyPI version](https://badge.fury.io/py/amundsen-databuilder.svg)](https://badge.fury.io/py/amundsen-databuilder)
-[![Build Status](https://api.travis-ci.com/lyft/amundsendatabuilder.svg?branch=master)](https://travis-ci.com/lyft/amundsendatabuilder)
+[![Build Status](https://api.travis-ci.com/amundsen-io/amundsendatabuilder.svg?branch=master)](https://travis-ci.com/lyft/amundsendatabuilder)
```

Comment:
Doesn't look like api.travis-ci.com/amundsen-io is real yet, I know @feng-tao and @jinhyukchang are fixing that, so let's make sure that's fixed and this link is live, before we merge this.

Reply:
it's real now!